python3/vimspector/debug_session.py

```diff
@@ -1410,6 +1413,8 @@ def _PrepareAttach( self, adapter_config, launch_config ):
       if attach_config[ 'pidSelect' ] == 'ask':
         prop = attach_config[ 'pidProperty' ]
         if prop not in launch_config:
+          # NOTE: We require that any custom picker process handles no-arguments
+          # as well as any arguments supplied in the config.
           pid = _SelectProcess()
```

Comment:
Is this meant to be passed something (maybe on a future commit in before accepting the PR)? If not, then why does the definition take `*args`?